src/routes/dashboard/header.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { icons } from "../../utils";
+import { useCurrentUser } from "../../hooks/useCurrentUser";
+
+export const Header: React.FC = () => {
+
+  const { user } = useCurrentUser();
+
+  function hover(e: React.MouseEvent<HTMLImageElement>, image: string): void {
+    (e.currentTarget as HTMLImageElement).src = image;
+  }
+
+  return (
+    <div className="flex justify-between bg-lightGray px-4 py-5">
+      <h2 className="font-text text-section-header font-medium leading-[32.8px]">
+        Hi, {user?.name || "guest"}!
+      </h2>
+      <div className="flex gap-4">
+        <Link to="/insights">
+          <img
+            src={icons.insights}
+            onMouseOver={e => hover(e, icons.insightsHover)} 
+            onMouseOut={e => hover(e, icons.insights)}
+          />
+        </Link>
+        <Link to="/notifications">
+         <img
+            src={icons.notification}
+            onMouseOver={e => hover(e, icons.notificationHover)} 
+            onMouseOut={e => hover(e, icons.notification)}
+          />        
+        </Link>
+        <Link to="/settings">
+          <img
+            src={icons.avatar}
```

Comment:
I don't see where we save an avatar for each user?

Reply:
Yeah. Seems like it's not in the backend. 
I was thinking it would be like WTWR where the user will have an avatar they can upload. 
Coding it now to anticipate a future update to the backend might be a good idea. 
Something to bring up to Josh when you get the chance.